### PR TITLE
Add User Style Attribute

### LINKED ISSUE #43: Modifying User migration

### PR DESCRIPTION
**Pull request by michaeljevans**
#### What's this PR do?
Add `style` attribute to User
#### Where should the reviewer start?
Bulk of changes occur in `code/api/src/migrations` and `code/api/src/modules/users`
#### How should this be manually tested?
Tests are complete for User queries; User mutations are working, but I'm not sure how to do teardown yet

Queries and mutations can be tested manually in the GraphiQL interface
#### Any background context you want to provide?
Add `style` to User so that it can be easily referenced; `style` is assigned after a user takes a survey
#### What are the relevant tickets?
closes #43 - Modifying User migration
### New Feature Submissions:
* [x] Have you added an explanation of what your changes do?
* [x] Have you linted your code locally prior to submission?
### Testing:
* [x] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] All new and existing tests passed?

### CODE REVIEW

**Comment by GreysonElkins** [comment]
Heads up @michaeljevans - depending on who's PR gets merged first (#61), the other may have to re-merge and re-rebase afterwords. Not sure tho?

File: code/api/src/modules/product/resolvers.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ export async function getRelated(parentValue, { productId }) {
       id: { [models.Sequelize.Op.not]: productId }
     },
     limit: 3,
-    order: [[models.Sequelize.fn('RAND')]] // mock related products by showing random products
+    order: [[models.Sequelize.fn('RANDOM')]] // mock related products by showing random products
```
Comment (Joshua-Tukman) [inline code/api/src/modules/product/resolvers.js:41]: Good catch here!

**Review by Joshua-Tukman — APPROVED** [review]
I had fun working with you on implementing the user style. I've got some more tests we can add to the project

File: code/api/src/modules/product/resolvers.js (hunk 1 above)
Comment (sciencefixion) [reply code/api/src/modules/product/resolvers.js:41]: Yeah, this still makes me laugh. XD

**Review by sciencefixion — APPROVED** [review]

**Pull request merged**